examples/ios/src/feedback.rs

```diff
@@ -34,7 +34,9 @@ pub fn run_example() -> Result<(), coreaudio::Error> {
 
     let format_flag = match SAMPLE_FORMAT {
```

Comment:
Per ios build fail, looks like `I24` needs to be handled

```
error[E0004]: non-exhaustive patterns: `I24` not covered
  --> src/feedback.rs:35:29
   |
35 |     let format_flag = match SAMPLE_FORMAT {
   |                             ^^^^^^^^^^^^^ pattern `I24` not covered
   | 
  ::: /Users/runner/work/coreaudio-rs/coreaudio-rs/src/audio_unit/sample_format.rs:11:5
   |
11 |     I24,
   |     --- not covered
   |
   = help: ensure that all possible cases are being handled, possibly by adding wildcards or more match arms
   = note: the matched value is of type `SampleFormat`
```

Reply:
Should be fixed now! Can we run the workflow again?